assets/airports/cyow.json

```diff
@@ -0,0 +1,537 @@
+                ["N45d42m14.00","W075d40m50.00"],
+                ["N45d42m28.00","W075d40m00.00"],
+                ["N45d42m41.00","W075d39m09.00"],
+                ["N45d42m52.00","W075d38m17.00"],
+                ["N45d43m00.00","W075d37m25.00"],
+                ["N45d43m07.00","W075d36m32.00"],
+                ["N45d43m12.00","W075d35m38.00"],
+                ["N45d43m15.00","W075d34m44.00"],
+                ["N45d43m16.00","W075d33m51.00"]
+            ]
+        }
+    ],
+    "fixes": {
+        "ADVOS":    ["N45d20m19.00", "W075d23m90.00"],
+        "ALIDO":    ["N45d44m43.00", "W075d40m77.00"],
+        "ALSET":    ["N45d36m62.00", "W074d30m00.00"],
+        "AMAGI":    ["N45d14m20.00", "W075d37m27.00"],
+        "APMUP":    ["N45d23m71.00", "W075d36m94.00"],
+        "ART":      ["N43d57m13.00", "W076d03m88.00"],
+        "BEMOG":    ["N46d09m04.00", "W075d34m35.00"],
+        "BOREG":    ["N45d14m57.00", "W075d36m43.00"],
+        "BUGSY":    ["N44d43m36.00", "W074d08m48.00"],
+        "CATOG":    ["N45d55m03.00", "W072d53m03.00"],
+        "CAVUR":    ["N45d11m12.00", "W075d29m27.00"],
+        "CYRIL":    ["N44d57m17.00", "W075d24m15.00"],
+        "DEMKO":    ["N45d10m45.00", "W075d39m97.00"],
+        "DUKTI":    ["N45d07m32.00", "W075d29m48.00"],
+        "DUNUM":    ["N45d34m25.00", "W075d49m08.00"],
+        "EBNAP":    ["N45d07m91.00", "W075d50m77.00"],
+        "ELSUB":    ["N44d27m70.00", "W077d11m18.00"],
+        "GABEV":    ["N45d28m80.00", "W075d52m80.00"],
+        "IGTAD":    ["N45d29m73.00", "W075d29m12.00"],
+        "ILOGI":    ["N45d29m63.00", "W075d48m39.00"],
+        "ILUBO":    ["N45d23m09.00", "W075d35m90.00"],
+        "KANIK":    ["N44d39m00.00", "W076d57m38.00"],
+        "KAREB":    ["N45d20m99.00", "W075d20m84.00"],
+        "KEMLU":    ["N45d24m21.00", "W075d50m96.00"],
+        "LANRK":    ["N44d56m58.00", "W076d23m28.00"],
+        "LITRI":    ["N45d21m83.00", "W075d33m88.00"],
+        "MEGNO":    ["N45d21m51.00", "W075d56m83.00"],
+        "MERKO":    ["N45d27m07.00", "W075d43m58.00"],
+        "MIVOK":    ["N44d21m60.00", "W077d35m34.00"],
+        "MODON":    ["N45d17m57.00", "W075d56m19.00"],
+        "NOVUX":    ["N45d12m57.00", "W075d40m88.00"],
+        "OBTAX":    ["N45d41m68.00", "W073d16m10.00"],
+        "OMTUN":    ["N45d22m69.00", "W075d59m11.00"],
+        "ONBAB":    ["N45d28m92.00", "W075d36m08.00"],
+        "ONDOB":    ["N45d41m62.00", "W076d26m25.00"],
+        "PETLU":    ["N45d30m81.00", "W075d50m69.00"],
+        "RAFNO":    ["N45d25m55.00", "W075d52m64.00"],
+        "REVAL":    ["N45d27m69.00", "W075d39m57.00"],
+        "RWY04":    ["N45d38m28.00", "W075d26m40.00"],
+        "RWY07":    ["N45d30m28.00", "W075d14m55.00"],
+        "RWY14":    ["N45d06m36.00", "W075d16m34.00"],
+        "RWY22":    ["N45d00m25.00", "W075d53m03.00"],
+        "RWY25":    ["N45d08m14.00", "W076d05m34.00"],
+        "RWY32":    ["N45d31m38.00", "W076d03m42.00"],
```

Comment:
```
"RWY14":    ["N45d06m36.00", "W075d16m34.00"],
"RWY32":    ["N45d31m38.00", "W076d03m42.00"],
```

I'm just now realizing that these fixes are in fact not over the runway for which they are named (that was my original assumption), but they are at the departure end of the runway for which they are named. So *really* the `RWY14` fix is located at the start of runway 32.

Is this a convention you've been using in other files?